### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

**Comment by colmsnowplow** [comment]
Apologies there were so many little cleanup pieces @jbeemster. Merged your changes into this branch but waiting for you to come back to merge and deploy, since we need to wait regardless. Looks good though!

**Comment by colmsnowplow** [comment]
@jbeemster I think this is now ready for final reveiw (& release if it's all good :) )

File: README.md [hunk 1]
```diff
@@ -1,49 +1,209 @@
...
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_iglu_config"></a> [iglu\_config](#output\_iglu\_config) | n/a |
+| <a name="output_loader_config"></a> [loader\_config](#output\_loader\_config) | n/a |
+| <a name="output_nsg_id"></a> [nsg\_id](#output\_nsg\_id) | ID of the network security group attached to the Transformer Server nodes |
+| <a name="output_user_data"></a> [user\_data](#output\_user\_data) | n/a |
+| <a name="output_vmss_id"></a> [vmss\_id](#output\_vmss\_id) | ID of the VM scale-set |
```
Comment (jbeemster) [inline README.md:179]: Three of these outputs are not represented in `outputs.tf` - is there a mismatch here or some code not pushed?

**Review by jbeemster — COMMENTED** [review]
All looking good - one minor comment but assuming example still launches and Snowflake is still getting loaded happy to approve!

File: README.md (hunk 1 above)
Comment (colmsnowplow) [reply README.md:179]: Same as the others, my debug files - fixed

**Pull request merged**